src/main/java/JettyLauncher.java

```diff
@@ -64,6 +68,11 @@ public static void main(String[] args) throws Exception {
 //        connector.setPort(port);
 //        server.addConnector(connector);
 
+        // Disabling Server header
+        Arrays.stream(server.getConnectors()).map(Connector::getConnectionFactories).flatMap(Collection::stream)
+                .filter(HttpConnectionFactory.class::isInstance).map(HttpConnectionFactory.class::cast)
+                .map(HttpConnectionFactory::getHttpConfiguration).forEach(config -> config.setSendServerVersion(false));
```

Comment:
I don't think that stream API is appropriate in this case. for loop like following is natural and much enough.
```java
for(Connector connector: server.getConnectors()){
    for(ConnectionFactory factory: connector.getConnectionFactories()){
        if(factory instanceof HttpConnectionFactory){
            ((HttpConnectionFactory) factory).getHttpConfiguration().setSendServerVersion(false);
        }
    }
}
```